functions/go/set-annotation/main.go

```diff
@@ -0,0 +1,226 @@
+// This file will be processed and embedded to pluginator.
```

Comment:
Can you add the License header for this file?

I saw some files in set-namespace and set-label don't have a license header. Can you add the correct header to these files? Feel free to do it in this PR or in a followup PR.

Reply:
This PR is a little bit large. Will do it separately.

Reply:
SGTM